site/content/docs/commands/svc-package.en.md

```diff
@@ -10,11 +10,13 @@ $ copilot svc package
 ## What are the flags?
 
 ```bash
+  -a, --app string          Name of the application.
   -e, --env string          Name of the environment.
   -h, --help                help for package
   -n, --name string         Name of the service.
       --output-dir string   Optional. Writes the stack template and template configuration to a directory.
       --tag string          Optional. The service's image tag.
+      --upload-resources    Optional. Whether to upload dependency resources (e.g., image that needs to build and push), so that the output of `copilot svc package` can be directly used for deployment.
```

Comment:
```suggestion
      --upload-resources    Optional. Whether to upload dependency resources (e.g., an image that needs to be built and pushed), so that the output of `copilot svc package` can be directly used for deployment.
```